Makefile

```diff
@@ -464,6 +466,10 @@ docker-build-kubeadm-control-plane: ## Build the docker image for kubeadm contro
 	$(MAKE) set-manifest-image MANIFEST_IMG=$(KUBEADM_CONTROL_PLANE_CONTROLLER_IMG)-$(ARCH) MANIFEST_TAG=$(TAG) TARGET_RESOURCE="./controlplane/kubeadm/config/default/manager_image_patch.yaml"
 	$(MAKE) set-manifest-pull-policy TARGET_RESOURCE="./controlplane/kubeadm/config/default/manager_pull_policy.yaml"
 
+.PHONY: docker-build-clusterctl
+docker-build-clusterctl: ## Build the docker image for clusterctl with output binary name as 'clusterctl'
+	DOCKER_BUILDKIT=1 docker build --build-arg builder_image=$(GO_CONTAINER_IMAGE) --build-arg goproxy=$(GOPROXY) --build-arg ARCH=$(ARCH) --build-arg package=./cmd/clusterctl --build-arg outbin=clusterctl --build-arg ldflags="$(LDFLAGS)" . -t $(CLUSTERCTL_IMG)-$(ARCH):$(TAG)
+
 .PHONY: e2e-framework
 e2e-framework: ## Builds the CAPI e2e framework
```

Comment:
Fyi, after this PR merges: https://github.com/kubernetes-sigs/cluster-api/pull/6452/files

You can rebase and add the clusterctl image to the list of images that should be promoted